Q/QD/bundled/patches/static_constants.patch

```diff
@@ -0,0 +1,48 @@
+diff --git a/include/qd/dd_real.h b/include/qd/dd_real.h
+index dda26ae..a968a0f 100644
+--- a/include/qd/dd_real.h
++++ b/include/qd/dd_real.h
+@@ -80,22 +80,22 @@ struct QD_API dd_real {
+   double _hi() const { return x[0]; }
+   double _lo() const { return x[1]; }
+ 
+-  const dd_real _2pi;
```

Comment:
The other way round, no? :thinking:

Reply:
Oops, just noticed also. Should be fixed